src/hotspot/share/nmt/virtualMemoryTracker.hpp

```diff
@@ -194,15 +194,37 @@ class VirtualMemorySummary : AllStatic {
  */
 class VirtualMemoryRegion {
  private:
```

Comment:
Style, pre-existing: Remove this private as it's private by default at the start.